templates/partials/configuration_forms.html

```diff
@@ -1,18 +1,45 @@
 <div class="col-sm-12">
-  <form id="forms-upload-form">
+  <form id="forms-upload-xform">
     <div>
       <fieldset>
-        <legend translate>Upload Forms</legend>
-        <div class="alert alert-danger alert-dismissible" role="alert" ng-show="error">
+        <legend translate>Upload XForm</legend>
```

Comment:
If you're going to change the translation key, switch to the new format. Maybe `xform.upload`?